docs/guide/custom-functions.md

```diff
@@ -59,6 +59,8 @@ class CountHF extends FunctionPlugin {
 ```
 ## Aliases
 
+Aliases are available since the **0.4.0** version.
+
```

Comment:
Maybe we should use a badge here as well? `<Badge text="v0.4.0"/>` This would create a convention and make it easier to notice features that were added later.

```suggestion
## Aliases <Badge text="v0.4.0"/>

```

Or

```suggestion
## Aliases

Aliases are available since the <Badge text="v0.4.0"/> version.

```

WDYT?